### ISSUE TITLE
Installing v4.0.0 through pip install fails

### ISSUE THREAD

**Issue by athril** [cmlibs/zinc#270]
Although the v4.0.0 was deposited on PyPI at https://pypi.org/project/cmlibs.zinc/ currently it can't be installed through 
`pip install cmlibs.zinc`. The installation fails with the following error:

```
ERROR: Could not find a version that satisfies the requirement cmlibs.zinc (from versions: none)
ERROR: No matching distribution found for cmlibs.zinc
```

**Comment by hsorby** [comment]
I am going to assume that this is for a GNU/Linux system?  If that is the case then no, we don't currently have pip support for `cmlibs.zinc`.  We do have wheels suitable for installing  with pip from the release page.

**Comment by athril** [comment]
Yes, this might be the case. Any plans to add support for GNU/Linux or MAC?
